graylog2-server/src/test/java/org/graylog/testing/mongodb/MongoDBInstance.java

```diff
@@ -49,27 +41,19 @@
  */
 public class MongoDBInstance extends ExternalResource implements AutoCloseable {
```

Comment:
From my point of view, we can also get rid of the unused `createForEachTest` method.

Reply:
:+1: